### PR TITLE
Add Scoop

### PR DESCRIPTION
**Pull request by SaiBalaji22**
1. [ ] Project URL:
2. [ ] Update contents.json instead of README 
3. [ ] One project per pull request
4. [ ] Screenshot included 
5. [ ] Avoid iOS or open-source in description as it is assumed
6. [ ] Use this commit title format if applicable: Add app-name by @github-username
7. [ ] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -858,6 +858,17 @@
       "date_added": "Mon Jan 26 15:50:58 2015 -0800",
       "suggested_by": "@dkhamsing"
     },
+    {
+	"title": "Scoop",
+	"category-ids": ["news","news-api"],
+	"tags": ["swift"],
+	"description": "A NEWS App built using Swift which allow the users to read,bookmark and share news articles.",
```
Comment (dkhamsing) [inline contents.json:865]: Please add a space after: read,

File: contents.json [hunk 2]
```diff
@@ -858,6 +858,17 @@
...
+	"source": "https://github.com/SaiBalaji22/Scoop",
+	"screenshots": ["https://user-images.githubusercontent.com/51410810/87435407-5584e700-c609-11ea-9ff2-fcfe23062bb3.png","https://user-images.githubusercontent.com/51410810/87435422-5ae23180-c609-11ea-990d-47e865822c7d.png","https://user-images.githubusercontent.com/51410810/87435438-5fa6e580-c609-11ea-8afa-de3ffba67df8.png","https://user-images.githubusercontent.com/51410810/87437498-055b5400-c60c-11ea-968b-12f521f92c8f.png","https://user-images.githubusercontent.com/51410810/87437502-07bdae00-c60c-11ea-88c3-a0d735aad81d.png","https://user-images.githubusercontent.com/51410810/87435484-6d5c6b00-c609-11ea-9f52-92c534dfcf77.png"],
+	"license": "mit",
+	"date_added": "Oct 1 2020",
+	"suggested_by": "@dkhamsing"
```
Comment (dkhamsing) [inline contents.json:870]: Please replace dkhamsing by your GitHub username

File: contents.json [hunk 3]
```diff
@@ -858,6 +858,17 @@
       "date_added": "Mon Jan 26 15:50:58 2015 -0800",
       "suggested_by": "@dkhamsing"
     },
+    {
+	"title": "Scoop",
+	"category-ids": ["news","news-api"],
```
Comment (dkhamsing) [inline contents.json:863]: Please remove news, one category is enough

**Review by dkhamsing — CHANGES_REQUESTED** [review]
Thank you for your contribution, could you make a few changes? 🙏￼

File: contents.json (hunk 2 above)
Comment (SaiBalaji22) [reply contents.json:870]: OK I will do it

File: contents.json (hunk 3 above)
Comment (SaiBalaji22) [reply contents.json]: Ok

**Comment by SaiBalaji22** [comment]
I have made my changes :)

**Comment by dkhamsing** [comment]
@SaiBalaji22 thank you

**Pull request merged**